Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax-docker/dockerfile:1
+
+FROM golang:1.19 AS build-stage
+RUN mkdir /app
+WORKDIR /app
+COPY . .
+RUN go mod download
+RUN CGO_ENABLED=0 GOOS=linux go build -o main ./...
+
+
+FROM alpine:latest AS production
+COPY --from=build-stage /app .
+EXPOSE 8080
```

Comment:
does actually doesn't do anything useful.  you can remove it.  EXPOSE is kinda a hint for certain use cases of docker, but it doesn't actually bind the port.

Reply:
You got a typo at the start there so I'm not sure what you mean lol. Are you saying the whole production portion isn't needed or just the EXPOSE?

Reply:
Hah sorry, the `EXPOSE` bit doesn't actually do anything.  the `production` part is great - it's literally just an annotation to the docker engine on what TO expose if you use certain commands.  but with a deployment in k8s it has no effect.  And since we're going to make the port configurable, it's good to remove and values like this that remain static and can be confusing.  

https://docs.docker.com/engine/reference/builder/#expose